modules/virt-vm-template-wizard-fields-web.adoc

```diff
@@ -0,0 +1,94 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/importing_vms/virt-importing-vmware-vm.adoc
+// * virt/vm_templates/virt-creating-vm-template.adoc
+
+// VM wizard includes additional options to VM template wizard
+// Call appropriate attribute in the assembly
+
+[id="virt-vm-template-wizard-fields-web_{context}"]
+ifdef::virtualmachine[]
+= Virtual machine wizard fields
+endif::[]
+ifdef::vmtemplate[]
+= Virtual machine template wizard fields
+endif::[]
+
+|===
+|Name |Parameter |Description
+
+ifdef::vmtemplate[]
+|Template
+|
+|Template from which to create the virtual machine. Selecting a template will automatically complete other fields.
+endif::[]
+
+|Name
+|
+|The name can contain lowercase letters (`a-z`), numbers (`0-9`), and hyphens (`-`), up to a maximum of 253 characters. The first and last characters must be alphanumeric. The name must not contain uppercase letters, spaces, periods (`.`), or special characters.
+
+ifdef::vmtemplate[]
```

Comment:
These seem to be the only differences between the VM and the Template wizard fields - and they seem to be marked up correctly for conditionals - but this is now a separate module. 
This means that the conditionals here aren't doing anything, but also we have two locations to update in the future.
I highly recommend not duplicating this table, but including these conditionalised rows in the virt-vm-wizard-felds-web.adoc
Again - the conditionals here look correct, so you should be able to just copy lines 30-43 and pasting them at line 30 in the virt-vm-wizard-fields-web.adoc <-- in fact it looks like lines 30-34 are already there ready to go.

Reply:
Updated virt-vm-wizard-fields-web.adoc and adding it to the assembly. Removing virt-vm-template-wizard-fields-web.adoc.